app/logic/tracy.py

```diff
@@ -91,6 +91,13 @@ def getDepartments(self):
         return STUPOSN.query.with_entities(STUPOSN.ORG, STUPOSN.DEPT_NAME, STUPOSN.ACCOUNT) \
                             .distinct().order_by(STUPOSN.DEPT_NAME).all()
 
+    def getDepartmentsFromName(self, deptName):
+        """
```

Comment:
You might consider correcting the name of the function. getDepartmentsFromName implies that you are getting multiple departments, which is not true.